ui/src/components/system/Certificates.vue

```diff
@@ -483,7 +483,15 @@
                 <strong>{{$t('certificates.error')}}.</strong>
                 {{letsEncryptCertificate.errorMessage}}
               </div>
-
+              <div class="form-group">
+                <div class="col-sm-3 control-label"></div>
+                <div class="col-sm-9">
+                  <button @click="addKnownDomains()" class="btn btn-default" type="button">
+                    <span class="fa fa-plus card-icon-def"></span>
+                    {{$t('certificates.add_Knowndomain')}}
+                  </button>
+                </div>
+              </div>
```

Comment:
I've had brief chat also with Edoardo: having this button above the form is a bit misleading.

Please move the button below the form, just on the right of the existing "Add domain" button.

Reply:
![Screenshot (23)](https://user-images.githubusercontent.com/3164851/101049935-4389ed80-3584-11eb-9a02-b9be84a79c36.png)